ui/pages/confirmations/components/simulation-details/simulation-details.tsx

```diff
@@ -94,48 +98,77 @@ const EmptyContent: React.FC = () => {
  *
  * @param props
  * @param props.children
+ * @param props.isTransactionsRedesign
  */
-const HeaderLayout: React.FC = ({ children }) => {
+const HeaderLayout: React.FC = ({ children, isTransactionsRedesign }) => {
   const t = useI18nContext();
+
   return (
     <Box
       display={Display.Flex}
       flexDirection={FlexDirection.Row}
       alignItems={AlignItems.center}
       justifyContent={JustifyContent.spaceBetween}
     >
-      <Box
-        display={Display.Flex}
-        flexDirection={FlexDirection.Row}
-        alignItems={AlignItems.center}
-        gap={1}
-      >
-        <Text variant={TextVariant.bodyMdMedium}>
-          {t('simulationDetailsTitle')}
-        </Text>
-        <Tooltip
-          interactive
-          position="top"
-          containerClassName="info-tooltip__tooltip-container"
-          tooltipInnerClassName="info-tooltip__tooltip-content"
-          tooltipArrowClassName="info-tooltip__top-tooltip-arrow"
-          html={t('simulationDetailsTitleTooltip')}
-          theme="tippy-tooltip-info"
-          style={{ display: Display.Flex }}
+      {isTransactionsRedesign ? (
+        <HeaderWithAlert />
+      ) : (
+        <Box
```

Comment:
Minor, would it be more readable to put the fallback into a local component such as `LegacyHeader`?

Reply:
Done